source/simulate/analyse-simulation.md

```diff
@@ -0,0 +1,92 @@
+# How to analyse a simulation
+
+When you run a simulation with the Python API, you might want to understand how the result was calculated and what intermediate variables and parameters has been taken into account.
+
+> To trace a simulation calculation with the Web API, please see [/trace endpoint documentation](openfisca-web-api/trace-simulation.md).
+
+
+## Call simulation tracer
+
+Let's say that you already have a `Simulation` object called `simulation` and, following [run a simulation](simulate/run-simulation.md) example, you tried to calculate the `housing_allowance` for a set of households.
+
+To call the `tracer`, you should activate the `trace` option with `simulation.trace = True`. This allows you log calculation steps and print them with `simulation.tracer.print_computation_log()`.
+
+Here is an example:
+
+```python
+# -*- coding: utf-8 -*-
+
+from openfisca_core.simulation_builder import SimulationBuilder
+from openfisca_country_template import CountryTaxBenefitSystem
+
+
+TEST_CASE = {
+    # ... whole test case ...
+}
+
+tax_benefit_system = CountryTaxBenefitSystem()
+simulation_builder = SimulationBuilder()
+simulation = simulation_builder.build_from_entities(tax_benefit_system, TEST_CASE)
+
+# activate the trace
+simulation.trace = True
+
+# call a calculation
+housing_allowance = simulation.calculate('housing_allowance', '2011-01')
+
+# print calculation steps
+simulation.tracer.print_computation_log()
+```
+
+> For more information on the `tracer` functions, see this [Tracer class interface](openfisca-python-api/tracer.html)
+
+## Analyse simulation steps
+
+If we call the tracer with the following `TEST_CASE`:
```

Comment:
You don't really "call" the tracer, as it doesn't define a `__call__` method 😛 
If we activate the trace? 
If we use the tracer?